[{"name": "test.test_configuration.ConfigurationTest.test_configuation", "status": "errored", "message": "User matching query does not exist.", "passed": false, "points": ["5.1.1", "5.1.2", "5.1.3", "5.1.4", "5.1.5"], "backtrace": ["  File \"/usr/lib/python3.8/unittest/case.py\", line 60, in testPartExecutor\n    yield\n", "  File \"/usr/lib/python3.8/unittest/case.py\", line 676, in run\n    self._callTestMethod(testMethod)\n", "  File \"/usr/lib/python3.8/unittest/case.py\", line 633, in _callTestMethod\n    method()\n", "  File \"/home/mikko/opiskelu/cybersecurity/mooc-securing-software-21/part5-20.configuration/test/test_configuration.py\", line 38, in test_configuation\n    response = c.post('/transfer/', {'to': 'alice', 'amount': 10})\n", "  File \"/home/mikko/.local/lib/python3.8/site-packages/django/test/client.py\", line 751, in post\n    response = super().post(path, data=data, content_type=content_type, secure=secure, **extra)\n", "  File \"/home/mikko/.local/lib/python3.8/site-packages/django/test/client.py\", line 407, in post\n    return self.generic('POST', path, post_data, content_type,\n", "  File \"/home/mikko/.local/lib/python3.8/site-packages/django/test/client.py\", line 473, in generic\n    return self.request(**r)\n", "  File \"/home/mikko/.local/lib/python3.8/site-packages/django/test/client.py\", line 719, in request\n    self.check_exception(response)\n", "  File \"/home/mikko/.local/lib/python3.8/site-packages/django/test/client.py\", line 580, in check_exception\n    raise exc_value\n", "  File \"/home/mikko/.local/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 47, in inner\n    response = get_response(request)\n", "  File \"/home/mikko/.local/lib/python3.8/site-packages/django/core/handlers/base.py\", line 181, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n", "  File \"/home/mikko/.local/lib/python3.8/site-packages/django/contrib/auth/decorators.py\", line 21, in _wrapped_view\n    return view_func(request, *args, **kwargs)\n", "  File \"/home/mikko/.local/lib/python3.8/site-packages/django/views/decorators/csrf.py\", line 54, in wrapped_view\n    return view_func(*args, **kwargs)\n", "  File \"/home/mikko/opiskelu/cybersecurity/mooc-securing-software-21/part5-20.configuration/src/pages/views.py\", line 16, in transferView\n    receiver = User.objects.get(username=request.GET.get('to'))\n", "  File \"/home/mikko/.local/lib/python3.8/site-packages/django/db/models/manager.py\", line 85, in manager_method\n    return getattr(self.get_queryset(), name)(*args, **kwargs)\n", "  File \"/home/mikko/.local/lib/python3.8/site-packages/django/db/models/query.py\", line 435, in get\n    raise self.model.DoesNotExist(\n"]}]